charters/PM/charter.md

```diff
@@ -0,0 +1,60 @@
+# PM Team Charter
```

Comment:
Is the term "PM" defined anywhere?

Reply:
In the DCP community - not to my knowledge. We have a _DCP PM_ meeting but our primary slack channel is dcp-project-mgmt.

It's unusual to have a PM role in open source communities, but Kubernetes has a SIG-PM that [defines PM](https://github.com/kubernetes/community/tree/master/sig-pm#overview) as: _**Program, project, and product management** in open source software communities ..._ which covers the bases.

Reply:
I think PM is a historical (~ from June 2017) shorthand for the project manager/product owner group who each manage the different parts of the DCP collaboration and help guide the strategy for the DCP

I don't think it needs to be defined as such (though I have used the first sentence from the description in the ingest support charter where I refer to the PMs but maybe a little bit of history wouldn't go as miss in the description

Reply:
+1 for having a PM definition up front - even if it is only to define what PM (Project Management? Product Management?) stands for

Reply:
I would simply and simply pick one - I'd prefer *Program Management* for DCP.